papers/yellow/voting.tex

```diff
@@ -45,6 +45,8 @@ \subsection{Parameters table}
 \hline
 4 & Maximum cumulative computational cost of a block & 1000000 & - & 16384 & - \\
 \hline
+5 & Token access cost (in nanoErgs) & 100 & - & - & - \\
+\hline
 \end{tabular}
```

Comment:
Please also describe cost rules somewhere in the Transactions section of yellow paper